Khan/perseus
# Avoid rendering legacy interactive graph when we really want mafs

Mark and I ran into an issue when testing these graphs and toggling between
different feature flag states.  When switching from the legacy interactive
graph to the Mafs version, the InteractiveGraph component wasn't remounted, so
the state of the legacy graph was preserved. This caused the line segment to
mysteriously disappear from a segment graph when the feature flag was switched
off!

To prevent similar gotchas in the future, this PR isolates the legacy graph code
in its own component, which is only rendered if the relevant Mafs feature flag
is turned off. This is a more accurate reflection of our intent.

Issue: none

## Test plan:

- View the interactive graph gallery (`yarn dev`)
- Toggle the "Segment" feature flag on.
- Move the line segment on the first graph.
- Toggle the "Segment" feature flag off.
- There should still be a line segment on the graph.
